swaybar/input.c

```diff
@@ -278,6 +398,10 @@ static void seat_handle_capabilities(void *data, struct wl_seat *wl_seat,
 		bar->pointer.pointer = wl_seat_get_pointer(wl_seat);
 		wl_pointer_add_listener(bar->pointer.pointer, &pointer_listener, bar);
 	}
+	if ((caps & WL_SEAT_CAPABILITY_TOUCH)) {
```

Comment:
Need to release `wl_touch` in case the seat looses the capability (just like the pointer, see https://github.com/swaywm/sway/pull/3711/files#diff-00a64d3193d21f96087133944068ca3eR393).